src/Http/Middleware/Authenticate.php

```diff
@@ -27,7 +27,9 @@ class Authenticate extends BaseMiddleware
      */
     public function handle($request, Closure $next)
     {
-        $this->authenticate($request);
+        if (! $this->auth->userExists()) {
```

Comment:
If we register:
```PHP
    public function register()
    {
        $this->app->singleton(\Tymon\JWTAuth\Providers\Auth\Illuminate::class, function ($app) {
            return new \Tymon\JWTAuth\Providers\Auth\Illuminate(
                $app[\Illuminate\Contracts\Auth\Factory::class]->guard('user')
            );
        });
    }
```
Then the user will exist in this level. If not, it will be authenticated as before.